### PR TITLE
Batch buttons macro

### PR DESCRIPTION
**Pull request by Lilykos**

### CODE REVIEW

File: inspire/base/templates/format/record/Holding_Pen_HTML_detailed_macros.tpl [hunk 1]
```diff
@@ -41,7 +41,9 @@
 
 
 {% macro get_abstract(record) %}
-    {% if record.abstracts %}
-       {{ record['abstracts'][0]['value'] }}
+    {% if record['abstract'] is mapping %}
```
Comment (jmartinm) [inline inspire/base/templates/format/record/Holding_Pen_HTML_detailed_macros.tpl]: Isn't it called `abstracts` in the data model?

Comment (jalavik) [inline inspire/base/templates/format/record/Holding_Pen_HTML_detailed_macros.tpl]: Correct, `abstracts`.

**Comment by jalavik** [comment]
Depends on https://github.com/inspirehep/invenio-workflows/pull/1/files

**Comment by jalavik** [comment]
Thanks merged as 71853530b4f20bf1c3674f53a58beb6c0def8be5

**Pull request closed without merging**